apps/observable-sample/src/cat/entities/cat.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Cat {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column()
+  breed: string;
+
+  @Column({ type: 'int' })
+  age: number;
+
+  constructor(name: string, breed?: string, age?: number);
+  constructor(name: string, breed: string, age?: number);
+  constructor(name: string, breed: string, age: number);
+  constructor(name?: string, breed?: string, age?: number);
+  constructor(name?: string, breed?: string, age?: number) {
+    this.name = name || '';
+    this.breed = breed || '';
+    this.age = age || NaN;
+  }
```

Comment:
_If_ we keep this constructor, why not do something like

```suggestion
  constructor(partial: Partial<Cat> = {}) {
    Object.assign(this, partial);
  }
```

Reply:
let's say the entity test is an extra